02-solidjs/README.md

```diff
@@ -0,0 +1,45 @@
+# Solidjs
+
+In this example we have implemented the same as the previous one but using Solidjs.
+
+We still use `JSX` but instead of having a React state with the products we use signals. In this case, we don't need to wrap the `handleChange` function with `useCallback` nor the `filtered` with `useMemo` because these functions will be created only once (even if we update the properties of that product).
```

Comment:
One advantage for _React_ developers is that SolidJS makes use of _JSX_, then following this example we will just replace _useState_ with _Signal_ and by doing that we don't need to use _useCallback_ _useMemo_ for this case, why? Because the component is only rendered once thus the functions will be created only once.